packages/wonder-blocks-button/__snapshots__/generated-snapshot.test.js.snap

```diff
@@ -399,6 +399,7 @@ exports[`wonder-blocks-button example 3 1`] = `
   <a
     className=""
     disabled={true}
+    href="https://khanacademy.org"
```

Comment:
For some reason I thought that you had to remove the `href` when disabling a link, but apparently that's not the case.

Reply:
Hm, if we're not removing the `href`, we should add `aria-disabled` like @somewhatabstract mentioned.

Reply:
Yes, we shouldn't be setting `disabled`. Instead, links should be disabled using `tabIndex="-1"` and `aria-disabled="true"`.

Reply:
@somewhatabstract I conflated `aria-disabled` and `disabled`.  Thanks for the clarification.  I'll make this change.